src/components/search-filters/filter-group.tsx

```diff
@@ -11,168 +11,184 @@ import theme from '@mdb/flora/theme';
-            <TypographyScale variant="body1" color="mark">
-                {filterString}
-            </TypographyScale>
-        );
-    })();
+            const filterString =
+                filterNames.length < 5
+                    ? filterNames.join(', ')
+                    : `${filterNames.slice(0, 5).join(', ')} +${
+                          filterNames.length - 5
+                      }`;
+            return (
+                <TypographyScale variant="body1" color="mark">
+                    {filterString}
+                </TypographyScale>
+            );
+        })();
 
-    return (
-        <div className={className}>
-            {!!title && (
-                <div onClick={onExpand}>
-                    <div sx={titleStyles}>
-                        <TypographyScale variant="body1">
-                            {title}
-                        </TypographyScale>
-                        <SystemIcon
-                            size="small"
-                            strokeWeight="medium"
-                            name={ESystemIconNames.CHEVRON_DOWN}
-                            sx={{
-                                transform: expanded ? 'rotate(180deg)' : null,
-                                transitionDuration: theme.motion.linkAnimation,
-                            }}
-                        />
+        return (
+            <div className={className}>
+                {!!title && (
+                    <div onClick={onExpand}>
+                        <div sx={titleStyles}>
+                            <TypographyScale variant="body1">
+                                {title}
+                            </TypographyScale>
+                            <SystemIcon
+                                size="small"
+                                strokeWeight="medium"
+                                name={ESystemIconNames.CHEVRON_DOWN}
+                                sx={{
+                                    transform: expanded
+                                        ? 'rotate(180deg)'
+                                        : null,
+                                    transitionDuration:
+                                        theme.motion.linkAnimation,
+                                }}
+                            />
+                        </div>
+                        {isMobile && !expanded && mobileFilterList}
                     </div>
-                    {isMobile && !expanded && mobileFilterList}
-                </div>
-            )}
-            {expanded && (
-                <div>
-                    <div sx={itemsStyles(title)}>
-                        {items
-                            .slice(0, showAll ? undefined : 5) // Show 5 to start, then all if they click "Show more"
-                            .map(filter => {
-                                if (filter.subItems && filter.subItems.length) {
+                )}
+                {expanded && (
+                    <div>
```

Comment:
Do we need this `div` here or can we swap it out for a fragment `<>`?

Reply:
So my intention was to have the filters and the "Show More" link to be enclosed in the same element in the DOM, but thinking about it now, it might make more sense to have them as siblings, so I'll put them in a fragment.